start.py

```diff
@@ -8,16 +11,56 @@
 from osfoffline.database import drop_db
 from osfoffline.gui.qt import OSFOfflineQT
 from osfoffline.utils.singleton import SingleInstance
+from osfoffline.tasks.notifications import Notification
+from osfoffline import settings
 
+from updater4pyi import upd_source, upd_core
+from updater4pyi.upd_iface_pyqt4 import UpdatePyQt4Interface
+from updater4pyi.upd_defs import Updater4PyiError
 
-def running_warning():
+
+logger = logging.getLogger(__name__)
+
+
+def running_warning(message):
     warn_app = QApplication(sys.argv)
-    QMessageBox.information(None, 'Systray', 'OSF-Offline is already running. Check out the system tray.')
+    QMessageBox.information(None, 'Systray', message)
     warn_app.quit()
 
 
 def start():
-    SingleInstance(callback=running_warning)  # will end application if an instance is already running
+    # will end application if an instance is already running
+    SingleInstance(callback=running_warning('OSF-Offline is already running. Check out the system tray.'))
+
+    # Check for updates first and give user a way to get new version
+    try:
+        swu_source = upd_source.UpdateGithubReleasesSource(settings.REPO)
+        swu_updater = upd_core.Updater(current_version=settings.VERSION,
+                                       update_source=swu_source)
+        swu_interface = UpdatePyQt4Interface(updater=swu_updater,
+                                             progname=settings.NAME,
+                                             ask_before_checking=True,
+                                             parent=QApplication.instance())
+    except Updater4PyiError as e:
+        logger.exception(e.updater_msg)
+        if 'Connection Error' in e.updater_msg:
+            Notification().error('Cannot check for updates because you have no Internet connection.')
+
+    min_version = None
+    # Then, if the current version is too old, close the program
+    try:
+        r = requests.get(settings.MIN_VERSION_URL)
+        min_version = r.json()['min-version']
+    except requests.exceptions.ConnectionError:
+        Notification().error('Check for minimun verion requirements for OSF-Offline failed '
```

Comment:
Typo: should be "minimum version".

Please also update language based on community feedback; this error message is not very helpful for the user.